src/index.html

```diff
@@ -11,8 +11,48 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Imac"
+      />
+      <div class="product-details">
+        <h2 class="product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-code">Product code: 195434</p>
+        <div class="flex-container">
+          <div class="stars">
```

Comment:
Add the modifier classes
```suggestion
          <div class="stars stars--4">
```